ee/tabby-ui/components/chat/chat.tsx

```diff
@@ -345,11 +380,18 @@ function ChatRenderer(
         }\n${'```'}\n`
       }
 
-      const newUserMessage = {
+      const newUserMessage: UserMessage = {
         ...userMessage,
         message: userMessage.message + selectCodeSnippet,
         // If no id is provided, set a fallback id.
-        id: userMessage.id ?? nanoid()
+        id: userMessage.id ?? nanoid(),
+        // If a temporary user message draft exists, use it's selectContext and activeContext
```

Comment:
Based on this implementation, only the message content and relevant client context are editable after the `edit` is clicked